README.md

```diff
@@ -33,7 +33,7 @@ package in the image.
 
 `openhpc_slurm_configless`: Optional, default false. If true then slurm's ["configless" mode](https://slurm.schedmd.com/configless_slurm.html) is used. **NB: Requires Centos8/OpenHPC v2.**
 
-`openhpc_munge_key_path`: Optional, default ''. Define a path for a local file containing a munge key to use, otherwise one will be generated on the slurm control node.
```

Comment:
It is a breaking change and is in a release - but I can't believe anyone is actually using it given we're not! Given that I thought it would make the code and testing so much easier to just remove it. It is definitely quite hard to get munge key handling right for all circumstances so the fewer code paths the better IMO.

Given we're pre v1 we're "ok" in semver terms to make a breaking change but I agree it's not ideal

If you want an nfs-munge key I'd say add a task (a pre-hook in slurm-appliance) to add it to the server) then this code will still work.

@jovial what do you think?

Reply:
I can get behind that reasoning. Agree that it would nice to remove some of the complexity around this.